### ISSUE TITLE
As a customer, I want to be able to see each section separately such as appetizers, lunch specials, meats, vegetarians, etc.

### ISSUE THREAD

**Issue by ghost** [zhoumiabc/Tofu-Kitchen-App#18]

**Comment by zhangda77** [comment]
I really go for it if I can pick up the meal based on different sections. I am not a vegetarian, but I have friend who is, so I can invite her to create her profile over there...

**Comment by shabbirw** [comment]
Yes, and one of the intentions in promoting Tofu Kitchen is to promote a healthy eating environment, so it is important to separate all types of meals on the menu page.
